Samples/Voronoi/README.md

```diff
@@ -0,0 +1,17 @@
+# Building
+In `build.sh`, change the `INCLUDE` variable to the directory where your SDL header files are
```

Comment:
could you add "Enter your SDL path here"? Perhaps check if it exists in the default path, if it does not, then ask for the input? 🤔